doc/scripting.md

```diff
@@ -123,7 +123,7 @@ Amass executes the `resolved` callback function after successfully resolving the
 
 ```lua
 function resolved(ctx, name, domain, records)
-    crawl(ctx, "https://" .. name))
+    crawl(ctx, "https://" .. name)
```

Comment:
The `crawl` API call also requires a `max` parameter that can be set to 0 in these examples